node/src/chain_spec.rs

```diff
@@ -277,6 +277,19 @@ pub mod mainnet {
 		properties.insert("tokenDecimals".into(), 12.into());
 		properties.insert("ss58Format".into(), 42.into());
 
+		let mut sudo_authority = vec![
+			get_account_id_from_seed::<sr25519::Public>("Charlie"),
+			get_account_id_from_seed::<sr25519::Public>("Dave"),
+			get_account_id_from_seed::<sr25519::Public>("Eve"),
+		];
+		sudo_authority.sort();
```

Comment:
Is it worth adding a comment as to why we sort? It may be useful for people that are unaware and this template is perhaps their first taste of Polkadot/Substrate.

Reply:
Yes sure.. Will do that!